dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/tcp/TCPUtils.java

```diff
@@ -0,0 +1,312 @@
+                }
+
+                if (retryCount > 0) {
+                    if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                        Tr.debug(tc, "attempt to bind again after a wait of " + timeBetweenRetriesMsec + "ms; "
+                                + retryCount + " attempts remaining" + " for " + config.getExternalName());
+                    }
+                    // recurse until we either complete successfully or run out of retries;
+                    try {
+                        Thread.sleep(timeBetweenRetriesMsec);
+                    } catch (InterruptedException x) {
+                        // do nothing but debug
+                        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                            Tr.debug(tc, "sleep caught InterruptedException.  will proceed.");
+                        }
+                    }
+                    open(framework, bootstrap, config, newHost, inetPort, openListener, retryCount - 1);
+                } else {
+                    if (config.isInbound()) {
+                        Tr.error(tc, TCPMessageConstants.BIND_ERROR, new Object[] { config.getExternalName(), newHost,
+                                String.valueOf(inetPort), openFuture.cause().getMessage() });
+                    } else {
+                        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                            Tr.debug(tc, TCPMessageConstants.BIND_ERROR, new Object[] { config.getExternalName(),
+                                    newHost, String.valueOf(inetPort), openFuture.cause().getMessage() });
+                        }
+                    }
+                }
+            }
+        });
+        return openFuture;
+    }
+
+    private static FutureTask<ChannelFuture> startHelper(NettyFrameworkImpl framework, AbstractBootstrap bootstrap,
+            TCPConfigurationImpl config, String inetHost, int inetPort, ChannelFutureListener openListener)
+            throws NettyException {
+    	if(framework.isStopping()){ // Framework already started and is no longer active
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(tc, "server is stopping, channel will not be started");
+            }
+            return null;
+        }else{
+            try {
+                return framework.runWhenServerStarted(new Callable<ChannelFuture>() {
+                    @Override
+                    public ChannelFuture call() {
+                        return open(framework, bootstrap, config, inetHost, inetPort, openListener,
+                                config.getPortOpenRetries());
+                    }
+                });
+            } catch (Exception e) {
+                if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                    Tr.debug(tc, "caught exception performing late cycle server startup task: " + e.getMessage());
```

Comment:
I think it's referring to having issues starting up a task after server startup. CHFW does something similar [here](https://github.com/OpenLiberty/open-liberty/blob/12f6257b2c5820a289e1afff3f4e5e16fbc1aada/dev/com.ibm.ws.channelfw/src/com/ibm/ws/tcpchannel/internal/TCPPort.java#L315-L317C47) but it's not the same message. This could be change if we want to, maybe `NettyFramework signaled- caught exception::` could be better to follow similar logging as CHFW